templates/index.html

```diff
@@ -56,6 +60,11 @@
 
   <script type="text/javascript">
 
+  // initialize bootstrap tooltip 
+  $(function () {
+    $('[data-toggle="tooltip"]').tooltip()
```

Comment:
I think we can place the `$('[data-toggle="tooltip"]').tooltip();` line in the `$(function () {` function / [IIFE](http://en.wikipedia.org/wiki/Immediately-invoked_function_expression)  below. They are / can be executed altogether :+1: